imagebuildah/util.go

```diff
@@ -55,7 +57,25 @@ func downloadToDirectory(url, dir string) error {
 	return nil
 }
 
-// TempDirForURL checks if the passed-in string looks like a URL.  If it is,
+func stdinToDirectory(dir string) error {
+	logrus.Debugf("extracting stdin to %q", dir)
+	r := bufio.NewReader(os.Stdin)
+	b, err := ioutil.ReadAll(r)
+	if err != nil {
+		return errors.Wrapf(err, "Failed to read from stdin")
+	}
+	reader := bytes.NewReader(b)
+	if err := chrootarchive.Untar(reader, dir, nil); err != nil {
+		dockerfile := filepath.Join(dir, "Dockerfile")
+		// Assume this is a Dockerfile
+		if err := ioutil.WriteFile(dockerfile, b, 0600); err != nil {
+			return errors.Wrapf(err, "Failed to write butes to %q", dockerfile)
```

Comment:
butes?

Reply:
Woops, that'll be fixed in the next push.